pkg/plugins/golang/v2/plugin.go

```diff
@@ -25,7 +25,7 @@ import (
 const pluginName = "go" + plugins.DefaultNameQualifier
 
 var (
-	supportedProjectVersions = []string{config.Version2, config.Version3Alpha}
+	supportedProjectVersions = []string{config.Version2, config.Version3}
```

Comment:
Hi @estroz, 

I think we need still support v3-alpha. So, instead of remove `Version3Alpha ` we need to add `Version3` and keep the supportability for both. WDYT?

Reply:
solved in another comment.